### PR TITLE
update graphql tag

### PR DESCRIPTION
**Pull request by Slashek**

### CODE REVIEW

File: marketplace_builder/views/pages/graphql.liquid [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (pavelloz) [inline marketplace_builder/views/pages/graphql.liquid]: ✂️

File: marketplace_builder/views/pages/tutorials/apis/persisting-part-of-third-party-api-response.liquid [hunk 2]
```diff
@@ -87,7 +87,7 @@ headers: '{
 
 ### Step 4: Create GraphQL mutation
 
-In the callback, the `execute_query` tag invokes a GraphQL mutation with the provided arguments. The query accepts two arguments: the `user_id`, the ID of the user we want to update, and the `third_party_api_id` which is the value we want to store. The content of the GraphQL mutation file:
+In the callback, the `graphql res =` tag invokes a GraphQL mutation with the provided arguments. The query accepts two arguments: the `user_id`, the ID of the user we want to update, and the `third_party_api_id` which is the value we want to store. The content of the GraphQL mutation file:
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/apis/persisting-part-of-third-party-api-response.liquid]: ?

**Review by pavelloz — APPROVED** [review]

**Comment by Slashek** [comment]
the test shouldn't pass now, as @godot is using graphql inside the loop to show N+1 vs `models` difference - re-triggering them

**Comment by Slashek** [comment]
Thanks @pavelloz missed those places!

**Pull request merged**